src/bigcurve_test.nr

```diff
@@ -535,24 +535,3 @@ fn test_msm() {
 }
 }
 }
-
-#[make_test(quote{BN254}, quote{BigNum<3, 254, BN254_Fq_Params>}, quote{BN254Scalar})]
-pub struct BN254GenTests {}
-#[make_test(quote{Vesta}, quote{VestaFr}, quote{VestaScalar})]
-pub struct VestaGenTests {}
-#[make_test(quote{Pallas}, quote{PallasFr}, quote{PallasScalar})]
-pub struct PallasGenTests {}
-#[make_test(quote{BLS12_377}, quote{BLS12_377Fr}, quote{BLS12_377Scalar})]
-pub struct BLS377GenTests {}
-#[make_test(quote{BLS12_381}, quote{BLS12_381Fr}, quote{BLS12_381Scalar})]
-pub struct BLS381GenTests {}
-#[make_test(quote{Secp256k1}, quote{Secp256k1Fr}, quote{Secp256k1Scalar})]
-pub struct Secp256k1GenTests {}
-#[make_test(quote{Secp256r1}, quote{Secp256r1Fr}, quote{Secp256r1Scalar})]
-pub struct Secp256r1GenTests {}
-#[make_test(quote{Secp384r1}, quote{Secp384r1Fr}, quote{Secp384r1Scalar})]
-pub struct Secp384r1GenTests {}
-#[make_test(quote{MNT4_753}, quote{MNT4_753Fr}, quote{MNT4_753Scalar})]
-pub struct MNT4GenTests {}
-#[make_test(quote{MNT6_753}, quote{MNT6_753Fr}, quote{MNT6_753Scalar})]
-pub struct MNT6GenTests {}
```

Comment:
This is just deleting all the tests. Instead can you change them to work on modules such as in the bignum library?

Reply:
Oh sorry I deleted the tests by mistake, should’ve double checked the PR. Thanks!